package.json

```diff
@@ -6,21 +6,21 @@
   "main": "index.js",
   "scripts": {
     "build": "run-s build:packages build:compile",
-    "build:compile": "run-p build:compile:es2015 build:compile:next",
+    "build:compile": "run-p --aggregate-output build:compile:es2015 build:compile:next",
```

Comment:
wow you really know this package well

Reply:
I looked that up some time ago because parallelization is nice, but it looked really ugly (and confusing) in the Jenkins logs (and it had a lot of nested tasks)